beta-src/src/App.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from "react";
 import "./assets/css/App.css";
 import Box from "@mui/material/Box";
 import map from "./assets/svg/map.svg";
+import WDModal from "./components/wd-modal/WDModal";
```

Comment:
we don't want to import this in App. this PR is about the component build, not implementation of the component